tests/test_arithmetics_protos.rs

```diff
@@ -108,7 +108,7 @@ impl PyNumberProtocol for InPlaceOperations {
         self.value |= other;
     }
 
-    fn __ipow__(&mut self, other: u32) {
+    fn __ipow__(&mut self, other: u32, _modulo: Option<u32>) {
```

Comment:
I managed to reproduce this segfault.

By changing this, it no longer segfaults.

```suggestion
    fn __ipow__(&mut self, other: u32, _modulo: u32) {
```

However, it now raises a TypeError:

```
TypeError: unsupported operand type(s) for ** or pow(): 'builtins.InPlaceOperations' and 'int'
```

Reply:
I'm using the following incantation to test, by the way.

```cargo test --test test_arithmetics --no-default-features --features full```

On Python 3.7, Windows.

Reply:
Thanks. I don't think the solution is to make it non optional..

Reply:
I agree but I hope this helps tracking down the issue :)